crates/common/consensus/src/blob_sidecar.rs

```diff
@@ -19,6 +20,30 @@ pub struct BlobSidecar {
     pub kzg_commitment_inclusion_proof: FixedVector<B256, U17>,
 }
 
+pub fn get_blob_sidecars(
+    signed_block: SignedBeaconBlock,
+    blobs: Vec<Blob>,
+    blob_kzg_proofs: Vec<KZGProof>,
+) -> Vec<BlobSidecar> {
+    let signed_block_header = compute_signed_block_header(signed_block.clone());
+
+    let mut blob_sidecars = vec![];
+
+    for (index, blob) in blobs.iter().enumerate() {
+        let blob_sidecar = BlobSidecar {
+            index: index as u64,
+            blob: *blob,
+            kzg_commitment: signed_block.message.body.blob_kzg_commitments[index],
+            kzg_proof: blob_kzg_proofs[index],
+            signed_block_header: signed_block_header.clone(),
+            kzg_commitment_inclusion_proof: FixedVector::default(),
+        };
+        blob_sidecars.push(blob_sidecar);
+    }
+
+    blob_sidecars
+}
```

Comment:
Can you implement this function on the SignedBeaconBlock implementation thinking more about this